frontend/src/components/UserSearch.vue

```diff
@@ -29,6 +29,7 @@
       </v-col>
       <v-col class="col-4">
           <v-btn id="search-button" class="primary" medium @click.native="searchUser('&search='+userSearchInput.replaceAll('\\','%5C'))">Search Users</v-btn>
+          <v-btn id="clear-search-button-basic" class="clear-search-button-basic" medium @click.native="clearSearchCriteria">Clear Search</v-btn>
```

Comment:
Please match the style of the existing "Clear search" button on the Advanced Search form:

Advanced search:
![image](https://user-images.githubusercontent.com/1767127/158265202-2c1c4d29-90c5-4d77-b92f-22e3f7c4aaa2.png)

Basic search:
![image](https://user-images.githubusercontent.com/1767127/158265244-16db69bb-0c90-41f8-9457-17f9492287ff.png)

I assume you can just use the existing CSS class used by the existing button, and then you won't need to write new CSS.